### PR TITLE
Blank no more

### PR DESCRIPTION
**Pull request by SirRichardFrancis**
## About The Pull Request

Removes "NeoTheology - Blank" from EOTP rotation. Parent of NT discs was made into medkit disk, and blank ~~now /nt_blank~~ is gone forever.
This PR is tested out, NT disks work as intended, with no design lists or resource cost altered.

https://github.com/discordia-space/CEV-Eris/issues/6281

## Why It's Good For The Game

Fixes cool.

## Changelog
:cl:
fix: empty disk no longer appearing in EOTP
/:cl:

### CODE REVIEW

File: code/game/objects/items/weapons/design_disks/NeoTheology.dm [hunk 1]
```diff
@@ -153,12 +153,18 @@
 	)
 
 /obj/item/computer_hardware/hard_drive/portable/design/nt
+	disk_name = "NeoTheology Armory - NeoTheologian Medkit"
```
Comment (Firefox13) [inline code/game/objects/items/weapons/design_disks/NeoTheology.dm:156]:
```suggestion
	disk_name = "NeoTheology Armory - NeoTheology Medkit"
```

File: code/game/objects/items/weapons/design_disks/NeoTheology.dm [hunk 2]
```diff
@@ -153,12 +153,18 @@
...
+	icon_state = "neotheology"
+	designs = list(
+		/datum/design/autolathe/firstaid/nt
+	)
+
+/obj/item/computer_hardware/hard_drive/portable/design/nt_blank
```
Comment (Firefox13) [inline code/game/objects/items/weapons/design_disks/NeoTheology.dm:162]: Why don't you remove it then?

Comment (SirRichardFrancis) [reply code/game/objects/items/weapons/design_disks/NeoTheology.dm:162]: ¯\_(ツ)_/¯

**Review by TheShown911 — APPROVED** [review]
Could have just made the proc that generates disks in the eotp use subtypesof instead of typesof

**Pull request merged**